api/app/src/routes/fhir-proxy.ts

```diff
@@ -38,11 +38,12 @@ const router = proxy("https://fhir.staging.metriport.com", {
     // const orgRegex = /"Organization\/(.+)"/g;
     // const orgReplace = `"Organization/urn:oid:$1"`;
     // eslint-disable-next-line no-useless-escape
-    const urlRegex = /https\:\/\/fhir\.staging\.metriport\.com/g;
-    const urlReplace = `https://api.staging.metriport.com`;
-    const updatedPayload = payloadString
-      // .replace(orgRegex, orgReplace)
-      .replace(urlRegex, urlReplace);
+    // const urlRegex = /https\:\/\/fhir\.staging\.metriport\.com/g;
+    // const urlReplace = `https://api.staging.metriport.com`;
+    const updatedPayload = payloadString;
+    // const updatedPayload = payloadString
+    //   // .replace(orgRegex, orgReplace)
+    //   .replace(urlRegex, urlReplace);
```

Comment:
Leaving the URL as it came from the FHIR server, and that being the original domain CW's CHA Broker called, was the last bit of change to make the Document Query of Document Contribution to work.

Now just waiting for them to debug the Document Retrieval - they're throwing 404 when we try to download the doc from them.